applications/FluidDynamicsApplication/custom_elements/compressible_navier_stokes.h

```diff
@@ -0,0 +1,579 @@
+
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Elisa Magliozzi
+//
+
+#if !defined(KRATOS_COMPRESSIBLE_NAVIER_STOKES) 
+#define  KRATOS_COMPRESSIBLE_NAVIER_STOKES
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/ublas_interface.h"
+#include "includes/variables.h"
+#include "includes/serializer.h"
+#include "utilities/geometry_utilities.h"
+#include "includes/cfd_variables.h"
+
+// Application includes
+#include "fluid_dynamics_application_variables.h" 
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+// TODO: UPDATE THIS INFORMATION
+/**this element is a 3D stokes element, stabilized by employing an ASGS stabilization
+* formulation is described in the file:
+*    https://drive.google.com/file/d/0B_gRLnSH5vCwZ2Zxd09YUmlPZ28/view?usp=sharing
+* symbolic implementation is defined in the file:
+*    https://drive.google.com/file/d/0B_gRLnSH5vCwaXRKRUpDbmx4VXM/view?usp=sharing
```

Comment:
Are these up to date? I could not open the drive files.

Reply:
also the generator does not work. I mean that after symbolic generation the files do not compile

Reply:
Everything should be updated and working now